zanshinsdk/client.py

```diff
@@ -275,6 +280,40 @@ def iter_organization_alerts(self, organization_id: Union[UUID, str],
                                                       page=page_number, page_size=page_size)
             yield from page.get('data', [])
 
+    def create_scan_target(self, organization_id: Union[UUID, str], kind: ScanTargetKind, name: str, **kwargs) -> Dict:
+        """
+        Create a new scan target in Zanshin for this organization
+        :param organization_id: the ID of the organization
+        :param kind: the Kind of scan target (AWS, GCP, AZURE)
+        :param name: the name of the scan target
+        :param kwargs: extra parameters for each kind of scan target
+        :param schedule: schedule in cron format
```

Comment:
This parameter is not present in the method. Either remove the docstring or add the parameter with a daily default value.